src/averageAges.js

```diff
@@ -15,6 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  let mens = people.filter(person => person.sex === 'm');
+
+  if (century) {
+    mens = mens.filter(men => Math.ceil(men.died / 100) === century);
+  }
+
+  const total = mens.reduce((sum, age) =>
+    sum + (age.died - age.born),
+  0
+  );
```

Comment:
total what?
Also - seems you use this logic in each function - why don't we create a helper for that?

Reply:
also, let's take a look at the checklist again, p.2